apollo-normalized-cache-sqlite/build.gradle.kts

```diff
@@ -26,6 +32,7 @@ configure<org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension> {
       dependencies {
         api(projects.apolloApi)
         api(projects.apolloNormalizedCacheApi)
+        api(projects.apolloNormalizedCache)
```

Comment:
Our setup is a bit weird and allows using `apolloNormalizedCacheSqlite` without `apolloNormalizedCache` which is a bit confusing.

`apolloNormalizedCache` is the one with the `apollo-runtime` dependency so it pulls some code but from the consumer side of things I don't really see a reason to use `apolloNormalizedCacheSqlite` without `apolloNormalizedCache`. I'm happy to revert and keep the current behaviour though